src/main/webapp/template.xhtml

```diff
@@ -19,7 +20,7 @@
                 <f:metadata>
                     <f:viewParam name="id" value="#{TemplatePage.templateId}"/>
                     <f:viewParam name="ownerId" value="#{TemplatePage.ownerId}"/>
-                    <f:viewParam name="editMode" value="#{TemplatePage.editMode}"/>
+                    <o:viewParam name="editMode" value="#{TemplatePage.editMode}"/>
```

Comment:
Why was this editMode viewParam removed on guestbook.xhtml, but converted to an OmniFaces component on this file? Does not seem to be utilized on that page, as far as I can tell.

Reply:
It's used in manageTemplates where you can edit a template and the edit button passes the edit mode in the url:

<h:outputLink value="/template.xhtml?id=#{template.id}&#38;ownerId=#{manageTemplatesPage.dataverse.id}&#38;editMode=METADATA">
                                                    <h:outputText value="#{bundle['dataset.manageTemplates.tab.action.btn.edit.metadata']}" />
                                                </h:outputLink>